<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
********************************************************************************
**
** Copyright (c) 2017 The University of Notre Dame
** Copyright (c) 2017 The Regents of the University of California
**
** Redistribution and use in source and binary forms, with or without modification,
** are permitted provided that the following conditions are met:
**
** 1. Redistributions of source code must retain the above copyright notice, this 
** list of conditions and the following disclaimer.
**
** 2. Redistributions in binary form must reproduce the above copyright notice, this
** list of conditions and the following disclaimer in the documentation and/or other
** materials provided with the distribution.
**
** 3. Neither the name of the copyright holder nor the names of its contributors may
** be used to endorse or promote products derived from this software without specific
** prior written permission.
**
** THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS &quot;AS IS&quot; AND ANY
** EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
** OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
** SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
** INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
** TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
** BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
** CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
** IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
** SUCH DAMAGE.
**
**********************************************************************************

// Contributors:
 </comment>
 <class>CWE_MainWindow</class>
 <widget class="QMainWindow" name="CWE_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFD Client Program</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="HeaderWidget" name="header">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="panelTabs" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="panelTabsLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="tab_panel_stack">
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="CWE_welcome_screen" name="tab_welcome_screen"/>
      <widget class="CWE_help" name="tab_help"/>
      <widget class="CWE_manage_simulation" name="tab_manage_and_run"/>
      <widget class="CWE_job_list" name="tab_jobs_page"/>
      <widget class="CWE_file_manager" name="tab_files"/>
      <widget class="CWE_Parameters" name="tab_parameters"/>
      <widget class="CWE_Results" name="tab_results"/>
     </widget>
    </item>
    <item>
     <widget class="FooterWidget" name="footer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionCopyright_and_License"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionCreate_New_Simulation">
   <property name="text">
    <string>Create &amp;New Simulation</string>
   </property>
  </action>
  <action name="actionManage_Simulation">
   <property name="text">
    <string>Manage and &amp;Run Simulation</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen_existing">
   <property name="text">
    <string>&amp;Open existing</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionAbout_CWE">
   <property name="text">
    <string>&amp;About SimCenter CWE</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Landing_Page">
   <property name="text">
    <string>&amp;Landing Page</string>
   </property>
  </action>
  <action name="actionManage_Jobs">
   <property name="text">
    <string>Manage and &amp;Download Files</string>
   </property>
  </action>
  <action name="actionManage_Remote_Jobs">
   <property name="text">
    <string>Manage Remote &amp;Jobs</string>
   </property>
  </action>
  <action name="actionTutorials_and_Help">
   <property name="text">
    <string>Tutorials and &amp;Help</string>
   </property>
  </action>
  <action name="actionManage_and_Download_Files">
   <property name="text">
    <string>Manage and &amp;Download Files</string>
   </property>
  </action>
  <action name="actionCopyright_and_License">
   <property name="text">
    <string>Copyright and License</string>
   </property>
  </action>
  <action name="actionInflow_parameters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Inflow parameters</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderWidget</class>
   <extends>QFrame</extends>
   <header>commonUI/HeaderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FooterWidget</class>
   <extends>QFrame</extends>
   <header>commonUI/FooterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_welcome_screen</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_welcome_screen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_help</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_help.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_manage_simulation</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_manage_simulation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_job_list</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_job_list.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_file_manager</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_file_manager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_Parameters</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_parameters.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWE_Results</class>
   <extends>QWidget</extends>
   <header>cwe_guiWidgets/cwe_results.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>CWE_MainWindow</receiver>
   <slot>menuExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopyright_and_License</sender>
   <signal>triggered()</signal>
   <receiver>CWE_MainWindow</receiver>
   <slot>menuCopyInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuExit()</slot>
  <slot>menuCopyInfo()</slot>
 </slots>
</ui>
